trie/leavesRetriever/leavesRetriever.go

```diff
@@ -35,125 +31,31 @@ func NewLeavesRetriever(db common.TrieStorageInteractor, marshaller marshal.Mars
 	}
 
 	return &leavesRetriever{
-		iterators:      make(map[string]common.DfsIterator),
-		lruIteratorIDs: make([][]byte, 0),
-		db:             db,
-		marshaller:     marshaller,
-		hasher:         hasher,
-		size:           0,
-		maxSize:        maxSize,
+		db:         db,
+		marshaller: marshaller,
+		hasher:     hasher,
+		maxSize:    maxSize,
```

Comment:
should we set any limit to `maxSize` parameter?

Reply:
I do not think that it is needed. We have other API requests that don't have a max limit. Anyway, this endpoint also has a time limit.